src/components/ProductCard/ProductCard.tsx

```diff
@@ -0,0 +1,64 @@
+import { useState } from 'react';
+
+import styles from './styles.module.scss';
+import blankIcon from '../../assets/images/icons/favorites-blank.svg';
+import filledIcon from '../../assets/images/icons/favorites-filled.svg';
+import { PrimaryButtons } from '../../enums/PrimaryButtons';
+import { FavoritesButton } from '../FavoritesButton/FavoritesButton';
+import { PrimaryButton } from '../PrimaryButton/PrimaryButton';
+
+export const ProductCard = () => {
+  const [clickedBuy, setClickedBuy] = useState(false);
+  const [clickedFav, setClickedFav] = useState(false);
+
+  return (
+    <article className={styles.card}>
+      <a href="#">
+        <img
+          src="public/img/phones/apple-iphone-13-mini/midnight/00.webp"
+          alt="Apple iPhone 14 Pro 128GB Silver (MQ023)"
+          className={styles.picture}
+        />
+      </a>
+      <h2 className={styles.title}>Apple iPhone 14 Pro 128GB Silver (MQ023)</h2>
```

Comment:
I think the card title should also be part of the link to the detailed information. You can see how it is implemented in other online stores with similar cards